service/json_schemas/const_config_io.schema.json

```diff
@@ -0,0 +1,39 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "id": "https://geopm.github.io/const_config_io.schema.json",
+    "title": "ConstConfigIOGroup config file specification",
+    "definitions": {
+        "signal_info": {
+            "type": "object",
+            "properties": {
+                "description": {
+                    "type": "string"
+                },
+                "units": {
+                    "enum": ["none", "seconds", "hertz", "watts", "joules", "celsius"]
+                },
+                "domain": {
+                    "enum": ["board", "package", "core", "cpu", "memory", "package_integrated_memory", "nic", "package_integrated_nic", "gpu", "package_integrated_gpu", "gpu_chip"]
+                },
+                "aggregation": {
+                    "enum": ["sum", "average", "median", "logical_and", "logical_or", "region_hash", "region_hint", "min", "max", "stddev", "select_first", "expect_same"]
+                },
+                "values": {
+                    "type": "array",
+                    "items": {
+                        "type": "number"
```

Comment:
Consider making it `["number", "null"]` or `["number", "string"]` if you want to allow some parts of a domain to hold no value. I think the latter is more consistent with what we allow for policies. Bonus if you use a pattern-matched string type that accepts variations of `"nan"`.

Reply:
After reviewing with Brad and Lowren, they were suggesting to simply not allow nulls or nans (i.e. enforce all values are provided). If there's a valid use case to allow nulls/nans, let's revisit.